src/test/java/com/work/management/EmployeeServiceTest.java

```diff
@@ -0,0 +1,46 @@
+package com.work.management;
+
+import static org.junit.Assert.assertEquals;
+
+import com.work.management.dto.EmployeeDto;
+import com.work.management.entity.Employee;
+import com.work.management.service.employee.impl.EmployeeServiceImpl;
+import javax.annotation.Resource;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.BeanUtils;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@ActiveProfiles("test")
+public final class EmployeeServiceTest {
+
+  @Resource
+  private EmployeeServiceImpl employeeService;
+
+  private static final Employee employee = Employee.builder().firstName("Mudit").lastName("Tanwar")
```

Comment:
all static final variables are constants..and all constants should be named with Capital letters and _ only...rename it to EMPLOYEE.

Reply:
remeber this rule, and this in your doc, this should not get repeated again...probably this is the 5th time I am pointing out this mistake.

Reply:
> remember this rule, and this in your doc, this should not get repeated again...probably this is the 5th time I am pointing out this mistake.

Documented